data/brands/amenity/parcel_locker.json

```
@@ -410,8 +424,8 @@
         "amenity": "parcel_locker",
         "brand": "easybox",
         "brand:wikidata": "Q114496224",
-        "operator": "Sameday",
-        "operator:wikidata": "Q114417096",
+        "operator": "Delivery Solutions EOOD.",
```

Small change to match the spelling, according to the [company registar](https://portal.registryagency.bg/CR/en/Reports/ActiveConditionTabResult?uic=206957949).

```suggestion
        "operator": "Деливъри Солюшънс",
```

---

@Dimitar5555 Thank you!

The full legal name of the company would be _Деливъри Солюшънс ЕООД_ then, right?

---

Yes, but we avoid company types in the operator field.

---

I've removed this tag completely due to other reasons: https://github.com/osmlab/name-suggestion-index/pull/10131#issuecomment-2480687657